README.md

```diff
@@ -68,6 +68,13 @@ Or, they can be passed to `prettier` as arguments:
 prettier --tab-width 4 --write '**/*.xml'
 ```
 
+### Whitespace
+
+The default for `xmlWhitespaceSensitivity` is `"strict"`. This means that your whitespace symbols are important, and should not be adjusted.
+
+If you work with modern xml workflows, you may want the `"ignore"` setting, as this will produce a standardized amount of whitespace.
```

Comment:
I'm not sure `modern xml workflows` makes sense in this context. There are plenty of modern xml use cases that have whitespace-sensitive nodes, including xhtml. I think I would change this to `If your XML files do not require whitespace sensitivity, you can use the `"ignore"` setting.

I'd also indicate in here that the reason it's `"strict"` by default is because that matches the XML specification. By default all whitespace is significant unless specified otherwise. Since prettier's contract is that it never changes the semantic meaning of your program, we pretty much have to roll with this as the default.

Reply:
Sure! I’m not an xml expert, I’ll defer to your terminology. I happen to never have worked with an XML flavor that demanded that kind of care re: whitespace, but knowing how other people use Prettier for other languages, and judging by the many dups, I hope this doc update will help others.

```suggestion
If your XML files do not require whitespace sensitivity, you can use the the `"ignore"` setting, as this will produce a standardized amount of whitespace.
Since whitespace is *sometimes* semantically important for XML, prettier has to default to `"strict"` in order to avoid potentially changing the meaning of your program.
```